src/index.html

```diff
@@ -223,11 +212,42 @@ <h1>Recherche utilisateur</h1>
       gapi.load('client:auth2', initClient);
     }
 
+    /**
+     *  Fetches Google API key and Client ID from local file (in .gitignore list)
+     *  Do not push to public repository. googleKeys.example.json contains example format.
+     *  Keys must be setup in Google Developer Console.
+     *  For details: https://developers.google.com/identity/protocols/oauth2/javascript-implicit-flow
+     */
+     async function fetchApiKeys(){
+        try {
+            const response = await fetch('./assets/googleKeys.json');
```

Comment:
That means the keys are publicly accessible for whoever has access to the server. In our case that's not an huge deal as the server machine is also the client, but that wouldn't be safe anymore if we decide to host this application elsewhere. Ideally accessing the google sheet would be a backend job, isolated from the frontend.